tools/component-builder/package.json

```diff
@@ -12,6 +12,8 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "dependencies": {
+    "@spectrum-css/tokens": "^13.0.5",
```

Comment:
Component builder was missing these dependencies but uses the packages in the scripts.